### ISSUE TITLE
`r2pm install r2frida` fails - should I use another command?

### ISSUE THREAD

**Issue by rustymagnet3000** [radare/radare2-pm#40]
`r2pm install r2frida` always fails on my mac.

radare 2 is installed and working.  r2pm version installed was `r2pm 1.3.0`

```
Makefile:1: config.mk: No such file or directory
./configure
checking build system type... x86_64-unknown-darwin
checking host system type... x86_64-unknown-darwin
checking target system type... x86_64-unknown-darwin
checking for working directories... current
using prefix '/usr/local'
checking for c compiler... gcc
checking for c++ compiler... g++
Using PKGCONFIG: pkg-config
checking pkg-config flags for r_core... yes
checking for curl... /usr/bin/curl
checking for wget... no
checking for git... /usr/bin/git
checking for xz... /usr/local/bin/xz
checking for node... no
error: This program is required.
make: *** [config.mk] Error 1
**ERROR: Build failed**
```

**Comment by rustymagnet3000** [comment]
looking inside the makefile, wget is not mandatory (as it falls back to cURL).  

I wonder if it is a Frida version issue.  I have version `9.1.1.18` but the makefile is looking for `9.1.1.16`?

**Comment by radare** [comment]
Just read the error. It says that you dont have node installed

**Comment by rustymagnet3000** [comment]
Thank @radare.  That was my suspicion.  Although `pip install node` seemed to succeed `which node` showed nothing was installed.  I suspect this is a local problem where some depedencies are owned by Root and some by the local user.  

Anyway, I got it working by downloading the full package from https://nodejs.org/  Then I get the proper result with `which node` = `/usr/local/bin/node`

I did have to run `sudo r2pm install r2frida`.   Again, think this was related to my previous error of installing Frida with a root owner.  I think.
